qa/automation/ScoringPortal.Tests.Scenarios/terenteva/sign_in_with_Google_account.feature

```diff
@@ -0,0 +1,91 @@
+		| Divider                                       |
+ 		| 'Dataset' text line                           |
+ 		| Divider                                       |
+ 		| Drag & drop form								|
+ 		| Username text field							|
+ 		| User profile icon								|
+ 		| Sign out button with sign out icon			|
+
+ Scenario: 1_01. 'Sign in with Google' button on the QSPP 'User NOT signed in' version is interactive and hyperlinked
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	When User drags cursor to the 'Sign in with Google' button
+	Then 'Sign in with Google' button is highlighted
+	And Cursor appearance is changed from standard to Link pointer
+	And Hyperlink address is shown at the bottom left corner of the page
+
+ Scenario Outline: 1_02. User is redirected to Google Login page
+ 	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User Google account state is <User account state>
+	When User clicks on 'Sign in with Google' button
+	Then User is redirected to Google Accounts Login page
+	And Google Accounts Login page is opened in the same browser tab
+
+	  Examples: 
+	  | User account state                                     |
+	  | User has no Google account                             |
+	  | User has a Google account, is not signed in the account |
+	  | User has a Google account, is signed in it, not signed in QSPP before from the device and browser used |
+
+ Scenario: 1_03. User goes straight from QSPP 'User not signed in' to QSPP 'User signed in'
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User has a Google account
+	And User is signed in this Google account
+	And User has signed in to QSPP before from the device and browser used
+	When User clicks on 'Sign in with Google' button
+	Then User is redirected to QSPP 'User signed in' version
+	And QSPP 'User signed in' version is opened in the same browser tab
+
+ Scenario Outline: 1_04. User is signed in QSPP after managing Google account
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User account state is <User account state>
+	And User has clicked on 'Sign in with Google' button on QSPP 'User not signed in'
+	And User was redirected to Google Accounts Login page
+	And Google Accounts Login page is opened in the browser tab
+	When User completes <Actions> according to <User account state>
+	Then User is redirected to QSPP 'User signed in' version
+	And QSPP 'User signed in' version is opened in the same browser tab
+
+	  Examples: 
+	  | User account state                                                     | Actions								 |
+	  | User has no Google account                                             | User registers an account and signs in  |
+	  | User has a Google account, is not signed in the account                | User signs in the account				 |
+	  | User has a Google account, is signed in it, not signed in QSPP before* | User chooses the account to proceed	 |
```

Comment:
what if user has 2 Google accounts and currently signed in both on the device?
Are these inputs testing sign in to app under test or authentication with accounts.google.com?

Reply:
Originally, I added a condition of user having more than one account, so user would choose on the Google page, which one to sign in with, but then I thought, that it would be a matter of testing Google functionality, so deleted it.

For me it is still important, to lead the way from 'not signed in' to 'signed in' through Google, not testing, how it works, supposing, it does, what it should do, but still highlighting that what you should do to proceed further depends on initial state of user Google account.

I think, we need a comment from Olga here. @shepelevaolga would you be so kind to provide a comment on this matter?

Reply:
In case of user having 2 active and signed in Google accounts you'd be redirected to the accounts.google.com page with a choice of account to be made, it is ore like BVA test option which is not considered. However, in my opinion you are testing Google here, not our requirement.

Reply:
If we wouldn't consider any actions on Google page, we would never get through to the page with all the functionality on the display. And it seems to be a part of user behavior than should lead a user to this functional page.